src/secretsManager.ts

```diff
@@ -0,0 +1,36 @@
+import AWS from 'aws-sdk';
+
+const { IS_OFFLINE, AWS_REGION, STAGE } = process.env;
+export default class SecretsManager {
+    static initialize() {
+        if (IS_OFFLINE === 'true') {
+            AWS.config.update({
+                region: 'us-west-2',
+                accessKeyId: process.env.ACCESS_KEY,
+                secretAccessKey: process.env.SECRET_KEY,
+            });
+        }
+    }
+
+    static async getIntegrationTransformUrl(): Promise<string> {
+        const ssm = new AWS.SSM();
+        let path = process.env.INTEGRATION_TRANSFORM_PATH ?? 'fake_path';
+        if (IS_OFFLINE === 'true') {
+            path = `fhir-service.integration-transform.${AWS_REGION}.${STAGE}.url`;
+        }
+        const data = await ssm
+            .getParameter({
+                Name: path,
+                WithDecryption: true,
+            })
+            .promise();
+
+        if (data.Parameter?.Value === undefined) {
+            console.error(`Unable to get INTEGRATION_TRANSFORM_PATH. Path: ${path}`);
+        }
+
+        return data.Parameter?.Value ?? '';
```

Comment:
Why are we returning an empty string? 

I think it's better to fail explicitly. Otherwise you'd only know about the error much later, once a persistence api call is made.

Reply:
Yeah, initially I was thinking about throwing an error here instead of returning an empty string. However, upon testing, it produced a confusing user experience.

When a request is made, the `FhirConfig` is initialized first, and then the routing layer is initialized. If we throw an error here, while the `FhirConfig` is being set up, the routing layer won't be initialized yet.

In testing this results in every request to the FHIR Server returning with a 200 response and an empty body.

Reply:
Once deployed to lambda, throwing errors will return a 502 response. It seems only `serverless-offline` will return a 200. Therefore, I've shifted to throwing errors in this class.